doc/api/inspector.md

```diff
@@ -108,6 +108,49 @@ with an error. [`session.connect()`] will need to be called to be able to send
 messages again. Reconnected session will lose all inspector state, such as
 enabled agents or configured breakpoints.
 
+## inspector.ws
+
+The Inspector can be interacted with via a websocket-based protocol. These APIs
+allow programatic control of the websocket, equivalent to use `--inspect*` CLI
+options.
+
```

Comment:
It can't be. As is, all this PR does (ATM) is allow instead of forcing `--inspect` to be provided as a CLI option, it allows node to startup, and then the js API calls the same C++ APIs called to turn on the listener that would have been called if `--inspect` had been provided. There is zero general purpose protocol or websocket capability exposed.